.github/generate_config_json.py

```diff
@@ -0,0 +1,60 @@
+
+import argparse
+import json
+import re
+from functools import reduce
+from schematic.schemas.generator import SchemaGenerator
+
+
+def get_args():
+    """Set up command-line interface and get arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--jsonld_path',
+                        required=True, help='path to model jsonld file')
+    parser.add_argument('-v1', '--service_version',
+                        default='', help='version of schematic')
+    parser.add_argument('-v2', '--schema_version',
+                        default='', help='version of data model')
+    parser.add_argument('-o', '--out_dir',
+                        default='.', help='directory to save result')
+    return parser.parse_args()
+
+
+def unCamel(x):
+    """Split camel-case string by ' ' delimiter."""
+    new = reduce(lambda a, b: a + (
+        (b.upper() == b and (len(a) and a[-1].upper() != a[-1]))
+        and (' ' + b) or b), x, '')
+    return new
```

Comment:
This function is used to convert schema name (camel case) to ' ' separated string, i.e:
```
BulkRNA-seqAssay --> Bulk RNA-seq Assay
ScRNA-seqLevel1 --> Sc RNA-seq Level1
```

The caveats is that it will not be able to split within capital letters, like `SRRSBiospecimen`. I am not good at python. @milen-sage Let me know if you have a better idea to convert schema name to display name.

Reply:
@rrchai schematic has a function that retrieves the display name based on a list of one or more schema labels: 

https://github.com/Sage-Bionetworks/schematic/blob/5a8670abb077a35630dfa2e7ac77f7bf9fb912bc/schematic/schemas/generator.py#L343

to get the mm_graph passed as arg to the function you can have

mm_graph = sg.se.get_nx_schema()

That way can avoid duplication of functionality. Otherwise your code snippet is not bad and it should work in most cases.